src/functions/get/handler.ts

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Maybe we should rename this folder from `get` to something like `getSwapProposal` or something like that so that it's future-ready in case we decide to create another `GET` request. What do you think?

Reply:
It's a good point. On [842846a](https://github.com/HathorNetwork/hathor-atomic-swap-service/pull/3/commits/842846a316a2e27bd629e80455a57c90c7f6d4a8) both folders were renamed with the suffix `SwapProposal` to keep the naming consistency.